docs/platform_management_plan/documentation_management.rst

```diff
@@ -12,14 +12,85 @@
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
 
-Documentation management
-------------------------
+.. document:: Documentation Management Plan
+   :id: doc__documentation_mgt_plan
+   :status: draft
+   :safety: ASIL_B
+   :tags: platform_management
+
+Documentation Management Plan
+-----------------------------
 
 Purpose
 +++++++
 
+The documentation management plan describes how documents are handled in the S-CORE project.
+
 Objectives and scope
 ++++++++++++++++++++
 
+Goal of this plan is to describe
+
+* which documents exist
+* which attributes and lifecycle they have
+* how they are reviewed
+
 Approach
 ++++++++
+
+.. gd_guidl:: Documentation
+   :id: gd_guidl__documentation
+   :status: valid
+   :complies: std_req__iso26262__support_39, std_req__iso26262__support_40, std_req__iso26262__support_41, std_req__iso26262__support_42, std_req__iso26262__support_43, std_req__iso26262__support_44
+
+Some of the work products of the S-CORE project are modelled specifically
+(e.g. the requirements and architecture have a specific set of attributes)
+Others are modelled as general documents (e.g. the plans which are part of the program management plan or the verification reports).
+
+This plan deals with these documents, which have the following manually set attributes:
+
+* Title: the name of the document (mandatory)
+* Unique Id: the name of the document (mandatory)
```

Comment:
Here it looks like that the Title and the Unique Id have the same wording. Do we have a 40 character limitation for the working or should we point to the naming conventions of Unique Ids? 
```suggestion
* Unique Id: Id following the naming pattern of the document Title
```